pkg/asset/installconfig/gcp/client.go

```diff
@@ -222,6 +225,31 @@ func (c *Client) GetProjects(ctx context.Context) (map[string]string, error) {
 	return projects, nil
 }
 
+// GetProjectRegions gets the regions that are valid for the project. An error is returned when unsuccessful
+func (c *Client) GetProjectRegions(ctx context.Context, project string) (map[string]string, error) {
+	svc, err := c.getComputeService(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	gctx := context.Background()
+	r, err := svc.Regions.List(project).Context(gctx).Do()
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to get regions for project %s", project)
+	}
+
+	regionsCP := make(map[string]string)
+	for _, region := range r.Items {
+		if value, ok := gcpv.Regions[region.Name]; ok {
+			regionsCP[region.Name] = value
+		} else {
+			regionsCP[region.Name] = ""
```

Comment:
Later on when you use this value, you skip the regions which are not already known by the installer. Any reason why not do this here already?